packages/js/src/utils/scope.test.ts

```diff
@@ -0,0 +1,82 @@
+import { DEFAULT_SCOPE_STRING, ScopeValue, withDefaultScopes } from './scope';
+
+describe('withDefaultScopes', () => {
+  describe('Expect withDefaultScopes method to append default scope values to the head of scope string', () => {
+    describe('undefined param', () => {
```

Comment:
Why do we have nested `describe` blocks?

Reply:
group the similar test cases